docs/sources/helm-charts/mimir-distributed/configure/configure-autoscaling.md

```diff
@@ -0,0 +1,194 @@
+---
+description: Learn how to configure Grafana Mimir autoscaling when using Helm.
+menuTitle: Configure autoscaling
+title: Configure Grafana Mimir autoscaling with Helm
+weight: 30
+---
+
+# Configure Grafana Mimir autoscaling with Helm
+
+{{< admonition type="warning" >}}
+Autoscaling support in the Helm chart is currently experimental. Use with caution in production environments and thoroughly test in a non-production environment first.
+{{< /admonition >}}
+
+This topic explains how to configure autoscaling for Mimir components using KEDA and Kubernetes Horizontal Pod Autoscaler (HPA).
```

Comment:
```suggestion
You can configure autoscaling for Mimir components using KEDA and Kubernetes Horizontal Pod Autoscaler (HPA).
```

Reply:
Rather than documenting what the documentation does, we can just tell users what they can do.